app/gui2/src/components/GraphEditor.vue

```diff
@@ -51,37 +56,67 @@ const nodeSelection = provideGraphSelection(graphNavigator, graphStore.nodeRects
 
 const interactionBindingsHandler = interactionBindings.handler({
   cancel: () => interaction.handleCancel(),
-  click: (e) => (e instanceof MouseEvent ? interaction.handleClick(e) : false),
+  click: (e) => (e instanceof MouseEvent ? interaction.handleClick(e, graphNavigator) : false),
 })
 
+function environmentForNodes(nodeIds: IterableIterator<ExprId>): Environment {
+  const nodeRects = [...graphStore.nodeRects.values()]
+  const selectedNodeRects: Iterable<Rect> = [...nodeIds]
```

Comment:
If the argument is expected to be the selected nodes, the argument name should reflect that.

Reply:
I've added a comment. This might not always be the selected nodes. Please let me know if the comment clarifies the intent enough.